msgpack.go

```diff
@@ -1,3 +1,6 @@
+//go:build !go_tarantool_msgpack_v5
+// +build !go_tarantool_msgpack_v5
+
```

Comment:
How does it work? If I add this build tag, the package will be linked with v5 instead of default v2 version?
Could you leave some link in CONTRIBUTING.md for how to use it and how it works, please.

Reply:
I've added a note in [README.md](https://github.com/tarantool/go-tarantool/blob/oleg-jukovec/gh-124-msgpack-v5/README.md#build-tags). But I think it's a good idea to add a section `How to run tests with tags` or somethind like this into `CONTRIBUTING.md`. Do you need a more detailed description in `CONTRIBUTING.md`?

Reply:
I think the proposed section will be enough. But some link to how it works in go with those comment-tags would be also helpful and informative.

Reply:
Please check changes README.md and CONTRIBUTING.md. Are there enough?

Reply:
Yes, I think so, thank you!